arch/risc-v/src/mpfs/mpfs_i2c.c

```diff
@@ -762,6 +729,7 @@ static int mpfs_i2c_reset(struct i2c_master_s *dev)
   if (ret != OK)
     {
       up_enable_irq(priv->plic_irq);
```

Comment:
if the init fails, why enable plic_irq? is there some error irq that needs capturing? If that is the case, shouldn't the irq be enabled already in init before doing the operation which might cause irq? If that is not the case, maybe the irq should just be left disabled, since the bus is not functional

Reply:
At the moment init() cannot fail, and because init resets the I2C bus there should not be any irq that needs capturing. So leaving irq disabled is OK here.